dace/frontend/python/preprocessing.py

```diff
@@ -403,7 +403,20 @@ def has_replacement(callobj: Callable,
     """
     from dace.frontend.common import op_repository as oprepo
 
+    # Nothing from the `dace` namespace needs preprocessing
+    mod = None
```

Comment:
This apparently doesn't work for `math`, at least inside a tasklet :(

Reply:
Fixed